openssl-license

```diff
@@ -0,0 +1,131 @@
+This notice is provided with respect to the OpenSSL toolkit, which may be
+included with the JRE and JDK.
+
```

Comment:
I think we should add words to the effect that if the user does not want to install openssl they can safely delete it from the jdk.